docs/04-operation-guides/troubleshooting/eventing/evnt-04-free-jetstream-storage.md

```diff
@@ -6,9 +6,10 @@ title: Eventing backend stopped receiving events due to full storage
 
 NATS JetStream backend stopped receiving events due to full storage.
 
-You see one of the following messages:
-- `507 Insufficient Storage` error from Event Publisher Proxy.
-- `no space left on device` from the Eventing backend.
+You observe the following behaviour in the Event Publisher Proxy(EPP):
```

Comment:
```suggestion
You observe the following message in the Event Publisher Proxy (EPP):
```

Reply:
IMO let's keep it as is, since the one of the points underneath is not referring to message (the HTTP code). what do you think ?